### PR TITLE
Updates WHOIS privacy to mention no auto-renew

### LINKED ISSUE #653: Explain that WHOIS does not renew automatically
[This customer](https://dnsimple.groovehq.com/tickets/98107) was not sure if WHOIS protection auto-renewed. I couldn't find any customer-facing documentation about [this SOP](https://dnsimple.atlassian.net/wiki/spaces/SUPPORT/pages/313032709/Whois+Privacy+auto-renewal).

Suggested at https://support.dnsimple.com/articles/whois-privacy/#renew

### PR DESCRIPTION
**Pull request by kayla-broussard**
Closes #653

Updates the WHOIS privacy article under the Renewal heading to explicitly state that we currently do not offer automatic renewal of WHOIS privacy, and that it must be renewed manually.

### CODE REVIEW

File: content/articles/whois-privacy.markdown [hunk 1]
```diff
@@ -123,6 +123,8 @@ Once you disable the WHOIS Privacy protection, the contact details of your domai
 
 ## Renew WHOIS Privacy {#renew}
 
+<info> There currently is no auto-renewal for WHOIS Privacy. It must be renewed manually. </info>
```
Comment (dallasread) [inline content/articles/whois-privacy.markdown:126]:
```suggestion
<info>
There currently is no auto-renewal for WHOIS Privacy. It must be renewed manually.
</info> 
```
With <info> on one line, I get: 
image

**Review by dallasread — APPROVED** [review]
👍 Pending my suggestion.

**Review by itsalyse — APPROVED** [review]
LGTM pending Dallas's suggestion 👍

File: content/articles/whois-privacy.markdown (hunk 1 above)
Comment (kayla-broussard) [reply content/articles/whois-privacy.markdown:126]: Ack, good catch 👍

**Pull request merged**